### PR TITLE
remove unused import in prettier

### PR DESCRIPTION
**Pull request by kahirokunn**
remove unused import in prettier layer is better then.
https://www.npmjs.com/package/prettier-plugin-organize-imports

### CODE REVIEW

**Comment by kahirokunn** [comment]
@msutkowski It would be nice to be able to simplify the engine code as much as possible, but how about covering imports with this?

**Comment by kahirokunn** [comment]
also removed unused queryArgs.

**Comment by msutkowski** [comment]
@kahirokunn Could you show us the before/after of the output of these changes regarding the queryArg? I'm personally not a fan of inserting something unused, then having to know that there is a plugin that strips it out - I'd just be confused when reading the code in the future :D

I'm thinking we should probably setup tests (perhaps using `.toMatchInlineSnapshot()`) for the output at this point to make sure we're observing these changes.

File: test/generated.ts [hunk 1]
```diff
@@ -51,37 +40,22 @@ export const api = createApi({
       }),
     }),
     getInventory: build.query<GetInventoryApiResponse, GetInventoryApiArg>({
-      query: (queryArg) => ({ url: `/store/inventory` }),
+      query: () => ({ url: `/store/inventory` }),
```
Comment (msutkowski) [inline test/generated.ts:43]: Nice! :)

**Review by msutkowski — COMMENTED** [review]
Ping me on discord if you can. I was about to start writing tests for expected output and considering doing a full e2e style test where we generated `msw` handlers doing the same type of code gen 🤯 . This looks good though, thank you!

**Comment by kahirokunn** [comment]
I love msw!
generate msw handlers for test is sounds good!

**Comment by kahirokunn** [comment]
@msutkowski I've revert the prettier guy, so if you like what this says, I'd be happy to incorporate this patch! (And I won't get scolded by eslint for the unused-vars rule anymore!

**Pull request merged**